src/components/Header/Header.component.tsx

```diff
@@ -18,10 +27,21 @@ export const Header = () => (
             </div>
             <div className={styles.rightIcons}>
                 <div className={styles.pageTabs}>
-                    <TextWithRedirect url={'/#'} text={'About'}></TextWithRedirect>
-                    <TextWithRedirect url={'/#'} text={'Products'}></TextWithRedirect>
+                    <button
+                        className={activePage === 'about' ? styles.activeButton : styles.inactiveButton}
+                        onClick={() => handleActivePage('about')}
+                    >
+                        About
+                    </button>
+                    <button
+                        className={activePage === 'products' ? styles.activeButton : styles.inactiveButton}
+                        onClick={() => handleActivePage('products')}
+                    >
+                        Products
+                    </button>
                 </div>
-                <IconWithRedirect className={styles.cartIcon} icon={images.cartIcon} url="/#"></IconWithRedirect>
+
+                <CartIcon counter={totalCart}></CartIcon>
```

Comment:
For better clarity, you can change the name of total amount of products in your cart. Ex: totalProducts, productsAmount ets. insted of counter